components/TextFields/src/Theming/MDCTextInputControllerOutlined+MaterialTheming.m

```diff
@@ -14,9 +14,13 @@
 
 #import "MDCTextInputControllerOutlined+MaterialTheming.h"
 
-#import <MaterialComponents/MaterialTextFields+ColorThemer.h>
```

Comment:
Can you remove the dependency in the BUILD/Podspec files?

Reply:
Ah, not yet in this case because the filled theming extension is still referring to it

Reply:
That makes sense. Thanks!  It would be helpful for reviewing to include a note like that in the commit message next time. 👍